graylog2-server/src/main/resources/swagger/css/screen.css

```diff
@@ -413,6 +413,21 @@ body #header {
     font-size: 14px;
 }
 
+.warning-box {
```

Comment:
If we use the `Alert` component this will not be necessary

Reply:
Sorry, ignore this, thought it was a react css file, not swagger, probably the best way to accomplish this